src/webrtc/callFeed.ts

```diff
@@ -47,26 +49,33 @@ export enum CallFeedEvent {
     LocalVolumeChanged = "local_volume_changed",
     VolumeChanged = "volume_changed",
     Speaking = "speaking",
+    VoiceActivityThresholdChanged = "voice_activity_threshold_changed",
```

Comment:
I can't see this ever being emitted - am I blind?

Reply:
Element-Call side. Used to set the threshold from the settings menu.

Reply:
Ah, of course!